src/main/java/com/auth0/client/mgmt/filter/CheckpointPaginationFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.auth0.client.mgmt.filter;
+
+public class CheckpointPaginationFilter extends BaseFilter {
```

Comment:
Is this class needed? I think we handle checkpoint pagination elsewhere without needing a specific class; maybe we should have one but that maybe should be a separate consideration. I don't see this class being used anywhere except for in tests?

Reply:
W have removed the CheckpointPaginationFilter class, as we are using PageFilter class.